io_scene_godot/structures.py

```diff
@@ -111,6 +112,7 @@ def __init__(self, entry_type, heading_dict=(), values_dict=()):
         self.entry_type = entry_type
         self.heading = collections.OrderedDict(heading_dict)
 
+        # NOTE: contents is unused. Remove?
```

Comment:
I 'd rather keep it, if we really need some custom writing to node, it could be a good prompt